classes/book.rb

```diff
@@ -12,5 +12,7 @@ def initialize(publisher, cover_state, publish_date)
 
   private
 
-  def can_be_archived?; end
+  def can_be_archived?
+    super() || cover_state == 'bad'
```

Comment:
- [ ] [REQUIRED] I believe that you should use the instance variable instead: `@cover_state` instead of `cover_state`

- [ ] [RECOMMENDED] I think the condition `@cover_state == 'bad'` deserves its own private method:
**Suggestion**
`def bad_cover_state? 
    @cover_state == 'bad'
end`

Reply:
Nice catch @Sboursen it's indeed `@cover_state` as for the second advice yea I do think that it's better to creat it's own method, I will work on that.